src/Message/Response.php

```diff
@@ -141,10 +141,11 @@ public static function html($html)
      * ```
      *
      * @param mixed $data
+     * @param int $status
      * @return self
      * @throws \InvalidArgumentException when encoding fails
      */
-    public static function json($data)
+    public static function json($data, $status = null)
```

Comment:
This should also work:

```suggestion
    public static function json($data, $status = self::STATUS_OK)
```

Reply:
Thank you!! yes you are right.